examples/tornjak/README.md

```diff
@@ -40,3 +40,39 @@ kubectl -n spire-server port-forward service/spire-tornjak-frontend 3000:3000
 You can now access Tornjak at [localhost:3000](http://localhost:3000).
 
 See [values.yaml](./values.yaml) for more details on the chart configurations to achieve this setup.
+
+## Tornjak and Ingress
+
+Obtain the Ingress subdomain (this process is cloud provider specific) and assign it
+to the `APP_SUBDOMAIN` env. variable, then run the deployment with your values files:
+
+```shell
+export APP_SUBDOMAIN=$(kubectl get dns cluster -o jsonpath='{ .spec.baseDomain }')
+```
+
+Update the `values-ingress.yaml` file with your subdomain.
+
+_Note: The location of the apps subdomain may be different in certain environments_
+
+Then run the deployment with your values files:
+
+```shell
+helm upgrade --install --namespace spire-server spire charts/spire \
+--values examples/production/values.yaml \
+--values examples/tornjak/values.yaml \
+--set spiffe-csi-driver.kubeletPath=/var/data/kubelet \
+--set spiffe-csi-driver.restrictedScc.enabled=true \
+--values examples/production/example-your-values.yaml \
+--values examples/tornjak/values-ingress.yaml \
```

Comment:
Would really rather the user not update tornjak/values-ingress.yaml with custom things, when they could/should just do example-your-values.yaml. Maybe reorder so that example-your-values.yaml is last and suggest that file be edited instead?